src/shared_modules/utils/zlibHelper.hpp

```diff
@@ -13,14 +13,23 @@
 #define _ZLIB_HELPER_HPP
 
 #include "customDeleter.hpp"
+#include "minizip/unzip.h"
+#include "stringHelper.h"
 #include <filesystem>
 #include <fstream>
 #include <memory>
+#include <stdexcept>
+#include <string>
+#include <vector>
 #include <zlib.h>
 
+#define KB (1024)
+#define MB (1024 * 1024)
+
 using ZFilePtr = std::unique_ptr<gzFile_s, CustomDeleter<decltype(&gzclose), gzclose>>;
 
-inline constexpr int BUF_LEN {16384};
+inline constexpr int GZ_BUF_LEN {16 * KB};
+inline constexpr int ZIP_BUF_LEN {100 * MB};
```

Comment:
I understand that this value is based on usage needs. But after doing some searching I found that the recommended values are between a range of 4 KB to 64 KB. Which is quite different from the current 100 MB set.

Could you test the performance with 64 KB?

Reply:
So, 64KB does it quite well. Tests made with a zip file of 51MB and a compressed file of 693MB:

![image](https://github.com/wazuh/wazuh/assets/42682247/5b01a498-c383-42ee-96a4-7ffb93764b22)

I've made a mistake by choosing 100MB. Changes to 64KB.